modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -81,4 +97,233 @@ enter the specific IP address you want and related information. For example:
+
+.{op-system} installer `core.inst` arguments for ISO or PXE
+
+[source,adoc]
+|===
+|Argument |Description
+
+a|`coreos.inst.install_dev`
+
+a|Required. The block device on the system to install to. While `/sda` only is allowed, it is recommended to use the full path, such as `/dev/sda`.
+
+a|`coreos.inst.image_url`
+
+a|Optional: Download and install the specified {op-system} image, overriding `coreos.inst.stream`.
+
+* This argument should not be used in production environments and is intended for debugging purposes only.
+
+* You must use this option if you were already using the previous `coreos-installer`.
+
+* While this argument can be used to install a version of {op-system} that does not match the live media, it is recommended that you instead use the image that matches the version you want to install.
+
+* If you are using `coreos.inst.image_url`, you must also use `coreos.inst.insecure`. This is because the bare-metal media are not GPG-signed for {product-title}.
+
+a|`coreos.inst.ignition_url`
+
+a|Optional: The URL of the Ignition config. If no URL is specified, no Ignition config will be embedded.
+
+a|`coreos.inst.platform_id`
+
+a| Optional: The Ignition platform ID of the platform the {op-system} image is being installed on. Default is bare metal. This option determines whether or not to request an Ignition config from the cloud provider, such as AWS or WMware. For example: `coreos.inst.platform_id=vmware`.
+
+a|`coreos.inst.save_partlabel`
+
+a|Optional: Comma-separated labels of partitions to preserve during the install. Glob-style wildcards are permitted. The specified partitions do not need to exist.
+
+a|`coreos.inst.save_partindex`
+
+a|Optional: Comma-separated indexes of partitions to preserve during the install. Ranges `m`-`n` are permitted, and either `m` or `n` can be omitted. The specified partitions do not need to exist.
+
+a|`coreos.inst.insecure`
+
+a|Optional: Permits the OS image to be unsigned, and installs without doing a gpgcheck. Must be used with `coreos.inst.image_url`.
+
+a|`coreos.inst.skip_reboot`
+
+a|Optional: The system will not reboot after installing. Once the install finishes, you will receive a prompt that allows you to inspect what is happening during installation. This argument should not be used in production environments and is intended for debugging purposes only.
+
+a|`ignition.config.url`
+
+a|Identifies the location of an Ignition config that you build to pass selected values to be consumed by the live installer. This argument can be used for adding disk partitioning or other features you want to have in place before the actual installation runs.
+
+|===
+
+[discrete]
+== `coreos-installer` options for ISO install
+
+The following table shows the options and subcommands you can pass to the `coreos-installer` command during an ISO live install.
+
+.`coreos-installer` options for ISO
+
+[source,adoc]
+|===
+|Option |Description
+
```

Comment:
I left out a couple of options that referenced FCOS (`stream` and `stream-base-url`). OK?